### ISSUE TITLE
No supported window manager found

### ISSUE THREAD

**Issue by poncovka** [rhinstaller/initial-setup#131]
If the initial setup cannot find a supported window manager, it quits with a message: `Initial Setup finished successfully, disabling`. That doesn't seem right.

**Comment by M4rtinK** [comment]
Yeah, that's not good. :P We should definitely reflect the effective error state in the message or ideally rather fall back to the TUI on the same graphical console.

**Comment by poncovka** [comment]
I was able to reproduce this error in our kickstart tests. The `initial-setup-gui` package requires `firstboot(windowmanager)`, but this dependency was resolved as `mutter` which is not supported by initial setup. Therefore, initial setup hit the `No supported window manager found` bug and happily quit.
